File: 프로그래머스/unrated/120902. 문자열 계산하기/문자열 계산하기.py
def solution(my_string):
    str = my_string.split(' ');
    answer = int(str[0]);
    
    for i in range(len(str)):
        if str[i] == '+': answer += int(str[i+1]);
        if str[i] == '-': answer -= int(str[i+1]);
    
    return answer

File: 프로그래머스/unrated/120902. 문자열 계산하기/문자열 계산하기.js
function solution(my_string) {
    const splited = my_string.split(' ');
    let answer = Number(splited[0]);
    
    for(let i = 0; i < splited.length; i++){
        if(splited[i] === '+'){
            answer = answer + Number(splited[i+1])
        }
        if(splited[i] === '-'){
            answer = answer - Number(splited[i+1])
        }
    }
    return answer
}